src/user/user.service.ts

```diff
@@ -128,8 +122,8 @@ export class UserService {
 
   async uploadProfilePicture(uid: number, image: Express.Multer.File): Promise<UploadProfileResponse> {
     const { uid: foundUid } = await this.findOne({ uid: uid }, { shouldExist: true });
-    if (image.mimetype !== 'image/png' && image.mimetype !== 'image/jpeg') {
-      throw new BadRequestException('Invalid image type');
+    if (!image || (image.mimetype !== 'image/png' && image.mimetype !== 'image/jpeg')) {
```

Comment:
cannot upload pdf?

Reply:
I think PDFs are best for documents and large photos, so not ideal for a profile picture.

Reply:
Hmmm I think u should discuss with @pointmekin